### DISCUSSION TITLE
Display images and gifs in the comments

### DISCUSSION THREAD

**Discussion by hyperstown** [Docile-Alligator/Infinity-For-Reddit#1210] [Duplicate Ideas] [answered]
Just like the title says.
Currently gifs and images are displayed as links. 
There should be an option to load and display an actual image just like in the web:
![image](https://user-images.githubusercontent.com/64496017/200645889-9e9eff81-0047-4bed-afb7-b8224e6d9bc4.png)

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #935, please your ideas there
